File: Hiroshima.API.Project/Hiroshima.Maas.Services/Utility/Helper/SearchParams.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Hiroshima.Maas.Services.Utility.Helper
{
    public class SearchParams
    {
        public string DeviceId { get; set; }
        public int TravellerId { get; set; }
        public int PageIndex { get; set; }
        public int PageSize { get; set; }
        public int Lang { get; set; }
        public DateTime? StartDateAndTime { get; set; }
        public DateTime? EndDateAndTime { get; set; }
    }
}

File: Hiroshima.Mobile.Project/src/shared/models/SearchParams.ts
import { ISearchParams } from "../interfaces";

/**
 * Search params model
 */
export class SearchParams implements ISearchParams{
   /**
     * User device id
     */
    deviceId?:string;
    /**
     * Current page index
     */
    pageIndex:number;
    /**
     * Current page size
     */
    pageSize:number;
    /**
     * Selected language name
     */
    lang:number;
    /**
     * Start date and time
     */
    startDateAndTime?:string;
    /**
     * End date and time
     */
    endDateAndTime?:string;
}